openfisca_core/tests/assets/parameters.yaml

```diff
@@ -0,0 +1,124 @@
+
+# Regular parameter values : meta + VALUES handling variation of its value in time
+- variable: smic_horaire_brut
+  description: SMIC horaire brut
+  format: float
+  type: monetary
+  VALUES:
```

Comment:
Just me transcribing caps locked XML... and I was thinking that the transformation type (here just VALUES, there BAREME and LIN-whatever) helps understand what the parameter is about quicker.